packages/devextreme/js/__internal/ui/html_editor/modules/m_resizing.ts

```diff
@@ -72,7 +72,7 @@ export default class ResizingModule extends BaseModule {
 
   _clickHandler(e) {
     if (this._isAllowedTarget(e.target)) {
-      if (this._$target === e.target) {
+      if (this._$target === e.target || !this._$resizeFrame) {
```

Comment:
I add this method because i noticed it fails on this [test](https://github.com/DevExpress/DevExtreme/blob/be545b66d7d8ad13d6b2c8656a09e851ad88fca5/packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/resizingIntegration.tests.js#L49)


reason: is that when we disable test we completely remove the resize frame thus it will generate an error on clickHandler when it executes img click on the test

Reply:
[fixing of a consequence]
If resizing is disabled this `_clickHandler` should not be called because this event listener is supposed to be unsubscribed. Now actually it's not detached because of a mistake [here](https://github.com/DevExpress/DevExtreme/blob/be545b66d7d8ad13d6b2c8656a09e851ad88fca5/packages/devextreme/js/__internal/ui/html_editor/modules/m_resizing.ts#L68). It should be updated this way:
```javascript
eventsEngine.off(this.quill.root, `.${MODULE_NAMESPACE}`);
```